python/ray/air/checkpoint.py

```diff
@@ -199,6 +199,38 @@ def __repr__(self):
         parameter, argument = self.get_internal_representation()
         return f"{self.__class__.__name__}({parameter}={argument})"
 
+    @property
+    def uri(self) -> Optional[str]:
+        """Return checkpoint URI, if available.
+
+        This will return a URI to cloud storage if this checkpoint is
+        persisted on cloud, or a local ``file://`` URI if this checkpoint
+        is persisted on local disk and available on the current node.
+
+        In all other cases, this will return None. Users can then choose to
+        persist to cloud.
```

Comment:
This was a little confusing to me at first. Maybe it can be clarified with something like:

```suggestion
        In all other cases, this will return None. To populate this value, you can persist this checkpoint to a URI with :meth:`~Checkpoint.to_uri`.
```